rootfs-builder/rootfs.sh

```diff
@@ -495,17 +553,38 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-		info "Pull Agent source code"
-		go get -d "${GO_AGENT_PKG}" || true
-		OK "Pull Agent source code"
-
-		info "Build agent"
-		pushd "${GOPATH_LOCAL}/src/${GO_AGENT_PKG}"
-		[ -n "${AGENT_VERSION}" ] && git checkout "${AGENT_VERSION}" && OK "git checkout successful"
-		make clean
-		make INIT=${AGENT_INIT}
-		make install DESTDIR="${ROOTFS_DIR}" INIT=${AGENT_INIT} SECCOMP=${SECCOMP}
-		popd
+		if [ "$RUST_AGENT" == "no" ]; then
+			info "Pull Agent source code"
+			go get -d "${GO_AGENT_PKG}" || true
+			OK "Pull Agent source code"
+
+			info "Build agent"
+			pushd "${GOPATH_LOCAL}/src/${GO_AGENT_PKG}"
+			[ -n "${AGENT_VERSION}" ] && git checkout "${AGENT_VERSION}" && OK "git checkout successful" || info "checkout failed!"
+			make clean
+			make INIT=${AGENT_INIT}
+			make install DESTDIR="${ROOTFS_DIR}" INIT=${AGENT_INIT} SECCOMP=${SECCOMP}
+			popd
+		else
+			# build rust agent
+			info "Build rust agent"
+			# The PATH /.cargo/bin is apparently wrong
+			# looks like $HOME is resolved to empty when
+			# container is started
+			source /root/.cargo/env
+			local -r agent_dir="$(basename ${RUST_AGENT_PKG} .git)/src/agent"
```

Comment:
I think the `.git` extension is optional and removing it from this script would simplify it (here and below).